fontc/src/lib.rs

```diff
@@ -504,33 +278,33 @@ pub fn create_workload(
     add_feature_ir_job(&mut workload)?;
     add_kerning_group_ir_job(&mut workload)?;
     add_glyph_ir_jobs(&mut workload)?;
-    add_glyph_order_ir_job(&mut workload)?;
+    workload.add(create_glyph_order_work());
 
     // BE: f(IR, maybe other BE work) => binary
 
-    add_feature_parse_be_job(&mut workload)?;
+    workload.add(FeatureParsingWork::create()); // TODO obey create_x_work convention
     add_feature_comp_be_job(&mut workload)?;
     add_glyph_be_jobs(&mut workload)?;
-    add_glyf_loca_be_job(&mut workload)?;
-    add_avar_be_job(&mut workload)?;
-    add_stat_be_job(&mut workload)?;
-    add_meta_be_job(&mut workload)?;
-    add_cmap_be_job(&mut workload)?;
-    add_fvar_be_job(&mut workload)?;
-    add_gvar_be_job(&mut workload)?;
-    add_head_be_job(&mut workload)?;
+    workload.add(create_glyf_loca_work());
```

Comment:
next step is some sort of general `Work` trait with a `create` fn or something..